File: examples/calc/include/calc.php
#!/usr/bin/env php
<?php
$c = isset( $argv[1] ) ? $argv[1] : 0;
$result = 0;

eval( "\$result = " . $c . ";" );

print 
"<update>
  <widget name=\"input\">.
    <property name=\"text\">
      <string>" . $result . "</string>
    </property>
  </widget> 
</update>";

?>

File: examples/calc/include/calc.py
#!/usr/bin/env python
import getopt, sys

opts, v = getopt.getopt(sys.argv[1:], "", [] )

r = eval( v[0] );

print "\
<update>\
  <widget name='input'>\
    <property name='text'>\
      <string>" + str( r ) + "</string>\
    </property>\
  </widget>\
</update>"
